libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -44,7 +44,10 @@ import {
 import { MultiInputMobileComponent } from './multi-input-mobile/multi-input-mobile.component';
 import { MultiInputMobileModule } from './multi-input-mobile/multi-input-mobile.module';
 import { MULTI_INPUT_COMPONENT, MultiInputInterface } from './multi-input.interface';
+import '@angular/localize/init';
```

Comment:
We're not using `@angular/localize` in `core`, please remove this import.

Reply:
fixed

Reply:
Please remove this import otherwise, app that consumes the library can't be built without `@angular/localize` but we don't push users to use the angular package for localization, they can choose any.